src/lang/id_ID.rc

```diff
@@ -86,7 +86,7 @@ IDR_MAINMENU MENU
         MENUITEM "Pengaturan &Fon...", ID_FONTS
         MENUITEM "Tentukan &Jalur...", ID_SETPATHS
         MENUITEM SEPARATOR
-        MENUITEM "&UI Language Select (UI语言选择)...", ID_CHOOSEUILANG
+        MENUITEM "Pilih Bahasa &UI", ID_CHOOSEUILANG

```

Comment:
Please don't translate this menu item.

Reply:
may i change the english one only? i can keep the japanese one.

Reply:
Revert it to the original. Not japanese but chinese.

Reply:
thanks